modules/tensor_mechanics/src/materials/ComputeAxisymmetricRZSmallStrain.C

```diff
@@ -34,7 +34,7 @@ ComputeAxisymmetricRZSmallStrain::initialSetup()
 Real
 ComputeAxisymmetricRZSmallStrain::computeStrainZZ()
 {
-  if (_q_point[_qp](0) != 0.0)
+  if (!MooseUtils::relativeFuzzyEqual(_q_point[_qp](0), 0.0))
```

Comment:
The `absolueFuzzyEqual()` function is more appropriate here since we're checking for the case where it is about zero.